docs/dietpi_tools.md

```diff
@@ -127,163 +129,186 @@ You can return to the **DietPi-Software** tool to make further changes at any ti
 
 ## Configuration
 
-`dietpi-config` {: #DietPi-Config }
+=== "DietPi configuration"
 
-: Configure various system settings, from display / audio / network to _auto start_ options.
+    Configure various system settings, from display / audio / network to _auto start_ options.  
+    Run `dietpi-config`.
 
-: ![DietPi-Config screenshot](assets/images/dietpi-config.jpg)
+    ![DietPi-Config screenshot](assets/images/dietpi-config.jpg)
 
-: **Display Settings**
+    **Display Settings**
```

Comment:
I think Markdownlint will complain when not using headers here, `#### Display Settings` should visually have the same result.

Reply:
Does Markdownlint complain if we follow a `####` behind a `##` without having a `###`?